src/azure-cli/azure/cli/command_modules/appservice/_appservice_utils.py

```diff
@@ -20,7 +20,18 @@ def _generic_site_operation(cli_ctx, resource_group_name, name, operation_name,
         return (operation(resource_group_name, name)
                 if extra_parameter is None else operation(resource_group_name,
                                                           name, extra_parameter))
-
     return (operation(resource_group_name, name, slot)
             if extra_parameter is None else operation(resource_group_name,
-                                                      name, extra_parameter, slot))
+                                                      name, slot, extra_parameter))
+
+
+def _generic_settings_operation(cli_ctx, resource_group_name, name, operation_name,
+                                setting_properties, slot=None, client=None, api_version=None):
+    client = client or web_client_factory(cli_ctx, api_version=api_version)
+    operation = getattr(client.web_apps, operation_name if slot is None else operation_name + '_slot')
+    print("settings_properties=")
+    print(setting_properties)
```

Comment:
please avoid using print() in azure cli and try to use logger https://github.com/microsoft/knack/blob/dev/docs/logging.md